CMakeLists.txt

```diff
@@ -157,6 +157,10 @@ list(APPEND DGL_SRC ${DGL_SRC_1})
 if(USE_CUDA)
   dgl_config_cuda(DGL_CUDA_SRC)
   list(APPEND DGL_SRC ${DGL_CUDA_SRC})
+
+  include(cmake/util/FindNccl.cmake)
```

Comment:
Is this nccl included in cuda package?

Reply:
No, by default NCCL is not part of the cuda package, but it does require cuda. Most frameworks will also require NCCL.